asciinema2gif

```diff
@@ -118,3 +118,5 @@ convert -delay 5 -loop 0 frames/*.png gif:- | gifsicle --colors=256 --delay=6 --
 cd - >/dev/null
 mv "${tempdir}/asciicast.gif" "${output_file}"
 echo ">> Generated in: ${output_file}"
+echo "cleaning up the frames before quiting"
+find "${tempdir}" -name \*.png -delete
```

Comment:
true i was just trying to fix a possible case of someone creating lots of gifs till the frame files take up a significant amount of space